microsoft-edge/progressive-web-apps-chromium/how-to/serviceworker.md

```diff
@@ -77,7 +77,7 @@ Within the `fetch` handler, you may control whether a request goes to the networ
 
 ## Push Notifications
 
```

Comment:
The paragraph that contains "Ensure that the JavaScript file that defines" is too long.  Break it midway.  Don't start sentence w/ "Ensure"; the examples in dictionaries including our official The American Heritage Dictionary use 'to ensure' in midsentence.  Change to "Make sure", at start of sentence.  MWSG removed its guidance on this, but I studied multiple dictionaries to confirm that it's not usual to begin a sentence with 'ensure' used as a directive.